core/src/main/scala/sbtorgpolicies/libraries.scala

```diff
@@ -45,44 +45,45 @@ object libraries {
     "catbird"                  -> "18.1.0",
     "cats"                     -> "1.0.1",
     "cats-effect"              -> "0.8",
-    "cats-mtl"                 -> "0.2.2",
+    "cats-mtl"                 -> "0.2.3",
     "circe"                    -> "0.9.1",
     "config"                   -> "1.3.2",
-    "coursier"                 -> "1.0.0",
+    "coursier"                 -> "1.0.2",
     "discipline"               -> "0.8",
-    "doobie"                   -> "0.5.0-M14",
+    "doobie"                   -> "0.5.0",
     "embedded-kafka"           -> "1.0.0",
     "embedded-redis"           -> "0.6",
     "export-hook"              -> "1.2.0",
-    "finch"                    -> "0.16.1",
-    "fs2"                      -> "0.10.0",
+    "finch"                    -> "0.17.0",
+    "fs2"                      -> "0.10.1",
     "fs2-cats"                 -> "0.5.0",
```

Comment:
We can remove this as fs2 is based cats since 0.10

Reply:
👍